src/modules/apps/file.ts

```diff
@@ -142,44 +115,35 @@ export function getLinkedDepsDirs(linkConfig : LinkConfig): string[] {
   return values(linkConfig.metadata)
 }
 
-export const getIgnoredPaths = (root: string): string[] => {
-  try {
-    return readFileSync(join(root, '.vtexignore'))
-      .toString()
-      .split('\n')
-      .map(p => p.trim())
-      .filter(p => p !== '')
-      .map(p => p.replace(/\/$/, '/**')).concat(defaultIgnored)
-  } catch (e) {
-    return defaultIgnored
-  }
+export const getIgnoredPatterns = async (root: string) => {
+  const ignoreFiles = ['.vtexignore', '.gitignore']
+  const ignoredPatterns = await Promise.all(map(async file => {
+    try {
+      const content = await readFile(join(root, file))
+      return content.toString().split('\n')
+    } catch (e) {
+      return []
+    }
+  }, ignoreFiles))
+  return ignore().add(reduce((x, y) => [...x, ...y], [], ignoredPatterns))
```

Comment:
I think you can use Ramda's `flatten` method here instead of `reduce`ing the array.